netbox_branching/__init__.py

```diff
@@ -18,7 +19,14 @@ class AppConfig(PluginConfig):
 
     def ready(self):
         super().ready()
-        from . import events, search, signal_receivers
+        from . import constants, events, search, signal_receivers
```

Comment:
I don't think signal_receivers is used

Reply:
This module needs to be imported to register the signal receivers (`record_change_diff()` and `validate_branch_deletion()`).